### DISCUSSION TITLE
Still active?

### DISCUSSION THREAD

**Discussion by LosOmatic** [pyalarmdotcom/alarmdotcom#423] [General]
Is this still being actively developed and maintained?

**Comment by elahd** [comment]
No for now. I haven’t had time to keep up with this project but plan to
pick it up again soon.

**Reply by timinski** [reply]
@elahd :+1:   Thank you for your work on this!   

All runs well for us save for the two SkyBells we have linked up with Alarm.com services.  Currently their doorbells only alert on our phones which we're not always next to.  I'd love to create an automation to also send an mp3 to our audio system or something similar but cannot get the SkyBells detected in any manner yet via your integration.

Looking forward to seeing you back here!
